app/controllers/application_controller.rb

```diff
@@ -4,7 +4,7 @@ class ApplicationController < ActionController::Base
   helper_method :current_admin, :current_user
 
   def current_user
-    @current_user ||= RegisteredUser.find(session[:user_id]) if session[:user_id]
+    @current_user ||= RegisteredUser.find(session[:registered_user_id]) if session[:registered_user_id]
```

Comment:
@case-eee we don't have a User model, it is named RegisteredUser, and it really just has an online/offline status and a has_many for google_credentials or our credentials.

When I write that out it sounds weird. Would love to discuss.